Scheduler/observing_blocks_maintel/BLOCK-T581.json

```diff
@@ -36,7 +36,7 @@
       "standard": true,
       "parameters": {
         "image_type": "FLAT",
-        "exp_times": "$run_exp_time",
+        "exp_times": 10,
```

Comment:
so, you are hard-coding the exposure time to 10 here... why don't you create a separate variable for these? this would allow us to have better control of the exposure times per filter....